docs/src/less/ts-runtime.less

```diff
@@ -10,9 +10,7 @@
 @ts-unit-double: (@ts-unit * 2);
 @ts-unit-triple: (@ts-unit * 3);
 @ts-width-aside: 320px;
-@ts-width-grails: (
-	@ts-width-aside - @ts-unit-double
-);
+@ts-width-grails: (@ts-width-aside - @ts-unit-double);
```

Comment:
Did Prettier do this? I thought it was disabled :weary:

Reply:
it might be in some of my IDEs...

Reply:
wait, actually I did it by hand so it looks like it's been written by people and not weird computers!